Source/Core/UICommon/DiscordPresence.cpp

```diff
@@ -82,6 +87,30 @@ static void HandleDiscordJoin(const char* join_secret)
 
   event_handler->DiscordJoin();
 }
+
+std::string ArtworkForGameId(const std::string& gameid)
+{
+  static const std::set<std::string> REGISTERED_GAMES{
+      "GALX01",
+  };
+
+  std::string region_neutral_gameid = gameid;
+  if (region_neutral_gameid.size() >= 4)
+  {
+    region_neutral_gameid[3] = 'X';
```

Comment:
The two last characters of a six-character game ID vary between regions in some cases (when a game has been published by different companies in different regions). If you want to make a game ID region neutral, I would suggest removing all characters except for the first three, like we do with game INIs.

Reply:
Fixed.